docs/content/docs/custom-resource/pod-template.md

```diff
@@ -125,4 +124,4 @@ arr1: [{name: a, p2: v2}, {name: c, p2: v2}]
 merged: [{name: a, p1: v1, p2: v2}, {name: b, p1: v1}, {name: c, p2: v2}]
 ```
 
-Merging by name can we be very convenient when merging container specs or when the base and override templates are not defined together.
+当合并容器规范或者当基础模板和覆盖模板没有一起定义时，按名称合并可以非常方便。
```

Comment:
```suggestion
当合并容器规格或者当基础模板和覆盖模板没有一起定义时，按名称合并可以非常方便。
```

The original English doc doesn't need the `we`, right?

If yes, we can remove it in this PR directly.

Reply:
yes

Reply:
You didn't update the english doc, right?

Reply:
Yes, sorry, I overlooked it. It's been fixed